lib/utils.js

```diff
@@ -14,6 +14,7 @@ var generateQueryString = function(query, method, credentials) {
   var searchIndex = query.searchIndex || 'All';
   var responseGroup = (query.responseGroup || 'ItemAttributes').replace(/,/g, '%2C');
   var condition = query.condition || 'All';
+  var merchant = query.MerchantId || 'All';
```

Comment:
The attribute name should start with a lowercase letter.
Also, why the default value is 'All' ? I think we should skip it if no value is supplied.